app/device/accelerometer.py

```diff
@@ -1,4 +1,82 @@
-
+from i2c import I2C
+from time import sleep
+from time import time
+import numpy as np
 class Accelerometer(object):
+
+    ADDRESS = 0x53
+    #REGISTERS
+    DEVID = 0x00
+    BW_RATE = 0x2C # Data rate and power mode control
+    POWER_CTL = 0x2D # Power-saving features control
+    DATA_FORMAT = 0x31
+    DATAX0 = 0x32
+    DATAX1 = 0x33
+    DATAY0 = 0x34
+    DATAY1 = 0x35
+    DATAZ0 = 0x36
+    DATAZ1 = 0x37
+
+
     def __init__(self):
-        print 'lol'
+        self.i2c = I2C(1, 0x53)
+        #set resolution to 16g
+        self.i2c.clear_bit(self.DATA_FORMAT, 1)
+        self.i2c.clear_bit(self.DATA_FORMAT, 0)
+        #right justify with sign extension
+        self.i2c.clear_bit(self.DATA_FORMAT, 2)
+        #set maximum resolution mode (4mg/LSB)
+        self.i2c.set_bit(self.DATA_FORMAT, 3)
+        #set data rate to 3200Hz
+        self.i2c.write_byte(self.BW_RATE, 0b00001111)
+        #setting the device to measure
+        self.i2c.set_bit(self.POWER_CTL, 3)
+
+
+    def read(self):
+
+        sensor_read = self.i2c.read_block(self.DATAX0,6)
+        #block_read returns a list of int
+
+        raw_data = np.int16([
```

Comment:
also modifying these register values should be in parse raw data...lets keep the responsibility of read just to fetching the appropriate regiters and not perforing operations on them